### DISCUSSION TITLE
Add project ID to project name in download-button addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7695] [Ideas]
As stated in the title, it would be useful having the project ID added to the project name when downloading the project (as it used to be in the old Scratch website). This makes easier to find the project on the website, as often the project name is not unique (think of the hundreds "3d raycaster" projects). This is useful, for example, if the project has additional resources (such as projects with server side code) that you are interested in, or if you want to find the author of a project with a very common title.

**Comment by sfederici** [comment]
I created a [scratchaddon](https://github.com/user-attachments/files/29974339/scratchaddons-pr-project-id-download.zip) using chatgpt to perform this task. It is a modification of a [chrome extension](https://github.com/user-attachments/files/29974329/scratch-download-id-extension.zip) I created with chatgpt to add this possibility to chrome. The extension works perfectly. I don't know about the addon.

**Comment by Niki182** [comment]
Nice idea

**Comment by sfederici** [comment]
Sorry, I wasn't aware this had already been added to ScratchAddons (by shift+clicking the "download" button) as in https://github.com/ScratchAddons/ScratchAddons/pull/7698. Thanks!
